2-Draft-Accepted/RFCxxxx-PowerShellGet-3.0.md

```diff
@@ -0,0 +1,476 @@
+
+`Find-Module` will be retained to provide compatibility with v2.
+
+### Installing resources
+
+`Install-PSResource` will mirror the parameter set for `Find-PSResource`.
+The cmd will only work for modules unless `-DestinationPath` is
+specified which works for all resource types.
+Other types will use `Save-PSResource` (see below).
+A `-Repository` parameter accepts a specific repository name or URL to the repository:
+
+```powershell
+Install-PSResource myModule -Repository 'https://mygallery.com'
+# or
+Install-Module myModule -Repository 'https://mygallery.com'
+```
+
+If `-Repository` is not specified and the same resource is found in multiple
+repositories, then the resource is automatically installed quietly from the
+trusted repository with the highest version matching the `-Version` parameter
+(if specified, otherwise newest non-prerelease version unless `-Prerelease`
+is used).
+If there are no trusted repositories matching the query, then the newest version
+fulfilling the query will be prompted to be installed from the highest priority
+repositories.
+If there are multiple repositories with the same priority level containing the same
+version, the first one registered is used and will be prompted to install.
+
+`-TrustRepository` can be used to suppress being prompted for untrusted sources.
+`-IgnoreDifferentPublisher` can be used to suppress being prompted if the publisher
+of the module is different from the currently installed version.
+`-Reinstall` can be used to re-install a module.
+
+A `-DestinationPath` parameter allows specifying the target directory instead
+of the default one.
+This will be in a different parameter set than `-Scope`.
+
+A `-Type` parameter will accept an array of types to allow filtering.
+
+A `-Version` parameter will accept a string of a [Nuget version range syntax](https://docs.microsoft.com/en-us/nuget/reference/package-versioning#version-ranges-and-wildcards).
+
+A `-Tags` parameter will accept a string array and will filter based on PSResource tags.
+
+A `-Filter` parameter will accept a string array and will filter based on all PSResource metadata (including resource descriptions).
+
+A `-Proxy` parameter will accept a uri.
+
+A `-ProxyCredential` parameter will accept a PSCredential
+
+A `-Credential` parameter will accept a PSCredential.
+
+An `-IncludeDependencies` switch will include dependent resources in the returned results.
+
+A `-NoClobber` switch will prevent installing modules that have the same cmdlets
+as a differently named module already on the system.
+
+A `-Quiet` switch will suppress progress information.
+
+`-Scope` with `AllUsers` and `CurrentUser` will work the same as it does in v2.
+Default is `CurrentUser`.
+
+If aliases are enabled `Install-Module` will call to `Install-PSResource -Type Module`.
+
+v3 will continue to use the versioning folder format as v2 in that it will be
+Major.Minor.Patch and not contain the semver prerelease label.
+

```

Comment:
It could add the pre-release label in 7.1 and continue to use the old format for everything previous. At least get support for the pre-release label in 7.1 so we have the option to adopt this later.